### PR TITLE
Added Coins list on landing page with chart

### PR DESCRIPTION
**Pull request by Smerly**
Added Coins list on landing page with chart

### CODE REVIEW

File: src/Components/CoinListComponent/EachCoin.js [hunk 1]
```diff
@@ -1,9 +1,214 @@
...
+        } else if (num >= 1.0e+3) {
+            return `${Math.floor(num / 1.0e+3)}K`
+        } else {
+            return num
+        }
+    }
+
+    const returnGreenOrRed = (data) => {
+        return data[data.length - 1][1] > data[0][1] ? '#4ade80' : '#dc2626'
+    }
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinListComponent/EachCoin.js]: these mini functions should be in a separate file called utils

File: src/Components/CoinListComponent/EachCoin.js [hunk 2]
```diff
@@ -1,9 +1,214 @@
...
+    const weekFromNow = []
+
+    // Make weekFromNow list days of the week according to the current day
+    let weekIndex = 0
+    daysInAWeek.forEach((each) => {
+        const indexedDate = new Date()
+        indexedDate.setDate(today.getDate() + weekIndex)
+        weekFromNow.push(daysInAWeek[indexedDate.getDay()])
+        weekIndex++
+    })
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinListComponent/EachCoin.js:33]: you can use map for this instead of for Each

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/Components/CoinListComponent/CoinList.js [hunk 3]
```diff
@@ -1,13 +1,80 @@
...
 
 
 function CoinList () {
-    const [coins, setCoins] = useState(undefined)
+    const [coins, setCoins] = useState('hi')
+    const [renderedCoin, setRenderedCoin] = useState([])
+
+    // This is for features later down the road
+    const updateCoinSort = (sortType) => {
+        setRenderedCoin(renderedCoin.sort((a, b) => {
```
Comment (mallharpreet) [inline src/Components/CoinListComponent/CoinList.js]: [...renderedCoin].sort.....

Comment (mallharpreet) [reply src/Components/CoinListComponent/CoinList.js]: sort is mutating the array

**Review by mallharpreet — APPROVED** [review]

**Pull request merged**